README.md

```diff
@@ -1,6 +1,9 @@
-![coffee](https://i.ibb.co/xHw5pkq/coffee.png)
 
-Type-safe, Easy To Use, <b>Deno Configuration</b>
```

Comment:
Oops, I just found out these extra `<b>` tags, this text should be in bold with markdown rules!

Reply:
ok no problem bro